docs/quickstarts/ratify-high-availability.md

```diff
@@ -2,7 +2,35 @@
 
 The default Ratify installation relies on a single Ratify pod processing all requests. For higher performance and availability requirements, Ratify can be set to run with multiple replicas and a shared state store.
 
-## Add Helm Chart Dependencies
+Ratify installation/upgrade for HA scenarios can be done via a `helmfile` or manual installation steps. Both options are outlined in this document.
+
```

Comment:
helmfile is still in its early stage of development which is versioned 0.x. I think it's totally fine to use helmfile in the quickstart, but I feel using `helmfile` in HA should be noted that `helmfile` is still an early stage (experimental).

Reply:
I have added a Note to the HA guide about this